tests/angular2/src/app/shared/sdk/services/custom/Models.ts

```diff
@@ -0,0 +1,35 @@
+/* tslint:disable */
+import { Injectable } from '@angular/core';
+import { User } from '../../models/User';
+import { Room } from '../../models/Room';
+import { Message } from '../../models/Message';
+import { ApplicationCredential } from '../../models/ApplicationCredential';
+import { UserCredential } from '../../models/UserCredential';
+import { UserIdentity } from '../../models/UserIdentity';
+import { Category } from '../../models/Category';
+import { Account } from '../../models/Account';
+import { RoomAccount } from '../../models/RoomAccount';
+import { Storage } from '../../models/Storage';
+import { Core } from '../../models/Core';
+
+@Injectable()
+export class ModelsApi {
```

Comment:
I like a lot the idea of providing the models through a single singleton service, actually it fits quite well with the refactor I'm doing in my side where now I'm providing the real time and storage classes as injectable services.